DiceKeys/Views/Screens/CustomeDiceDemo/CustomDiceFaceView.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  CustomDiceFaceView.swift
+//  DiceKeys
+//
+//  Created by Kevin Shah on 16/01/21.
+//
+
+import SwiftUI
+
+/// DiceFaceManager
+class DiceFaceManager: ObservableObject {
```

Comment:
In a reactive (SwiftUI) design style, all the derived values can be part of the view.